src/main/java/ApplicationDriver.java

```diff
@@ -102,10 +93,34 @@ private static boolean handleQueryInput(String input) {
                 }
                 break;
             case "5":
-                System.out.println("Auto-scheduling is not yet implemented"); // TODO implement this functionality
+                printTasks();
+                Task task = chooseTask();
+                success = controller.suggestTimeToUser(task);
+                if (success) {
+                    System.out.println("Event created from task");
+                } else {
+                    System.out.println("Failed to create event from task");
```

Comment:
It does not give information on why fail to create the event
Maybe use exception in the future so that it can deal with more cases?

Reply:
ok i added a todo for future reference